client/src/components/Answer/Answer.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+function Answer () {
```

Comment:
I love all of your straight forward components' names. Great job with naming conventions.